app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -117,6 +117,8 @@
     <string name="message_welcome">Bem-vindo, %s</string>
     <string name="message_removed">Mensagem removida</string>
     <string name="message_pinned">Pinou uma mensagem:</string>
+    <string name="message_muted">Usuário %1$s silenciado por %2$s</string>
+    <string name="message_unmuted">Usuário %1$s de-silenciado por %2$s</string>
```

Comment:
I think that we need a better translation here. @rafaelks How do you show the "unmuted" word in Portuguese from the iOS app?

Reply:
Yeah, that's up to debate... One contributor have fixed some typos in our PT-BR and we, for now, accepted this "fix". Previously we had "mutar" and "desmutar" which also doesn't exist in Portuguese.